crates/compiler/gen_llvm/src/llvm/build_list.rs

```diff
@@ -398,6 +398,7 @@ pub(crate) fn list_capacity<'ctx>(
     builder: &Builder<'ctx>,
     wrapper_struct: StructValue<'ctx>,
 ) -> IntValue<'ctx> {
+    // TODO: This won't work on seemless slices. Needs to return the len if the capacity is negative.
```

Comment:
🤔 should we make the Zig function `getCapacity` public and then have this call it directly?

Reply:
yeah, probably a good idea especially think I believe it can be optimized to just use bit manipulation to get the capacity. Will be nice to centralize improvements.